src/ol/source/OSM.js

```diff
@@ -76,7 +76,8 @@ class OSM extends XYZ {
       reprojectionErrorThreshold: options.reprojectionErrorThreshold,
       tileLoadFunction: options.tileLoadFunction,
       url: url,
-      wrapX: options.wrapX
+      wrapX: options.wrapX,
+      attributionsCollapsible: true
```

Comment:
Did you mean to initialize this with `false`?

Reply:
Yes, this was a misstake, it is not initialized to false